packages/oui-back-button/src/back-button.html

```diff
@@ -1,20 +1,26 @@
 <button ng-if="::!$ctrl.href && !$ctrl.state"
-        type="button"
-        class="oui-button oui-button_icon-alone"
-        ng-attr-aria-label="{{:: $ctrl.ariaLabel }}"
-        ng-attr-id="{{:: $ctrl.id }}"
-        ng-attr-name="{{:: $ctrl.name }}"
-        ng-click="$ctrl.onBtnClick()">
+    type="button"
+    class="oui-button oui-button_icon-alone"
+    ng-attr-aria-label="{{:: $ctrl.ariaLabel }}"
+    ng-attr-id="{{:: $ctrl.id }}"
+    ng-attr-name="{{:: $ctrl.name }}"
+    ng-click="$ctrl.onBtnClick()">
   <i class="oui-icon oui-icon-chevron-left" aria-hidden="true"></i>
```

Comment:
Don't you take the opportunity to change the `i` into `span` ? It shows the bad example

Reply:
Yes, some may have escaped my radar ;)